java/flight/src/main/java/org/apache/arrow/flight/FlightClient.java

```diff
@@ -99,7 +102,7 @@ public FlightClient(BufferAllocator incomingAllocator, Location location) {
   }
 
   /**
-   * List actions available on the Flight service.
+   * Lists actions available on the Flight service.
```

Comment:
I don't have the best grammar, but I thought it was fine before.. as in "Calling this method will list actions available..."  maybe it could go either way?

Reply:
Neither do I.  In this case the change is dictated by my interpretation of the style guide: https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

Reply:
Ok seems fine then